erdblick_app/app/view.component.ts

```diff
@@ -272,14 +271,14 @@ export class ErdblickViewComponent implements AfterViewInit {
         }
 
         // Apply additional highlight styles.
-        for (let [styleId, styleData] of this.styleService.styleData) {
+        for (let [_, styleData] of this.styleService.styleData) {
             if (styleData.featureLayerStyle && styleData.enabled) {
                 let visu = new TileVisualization(
                     resolvedFeature!.featureTile,
                     (tileKey: string)=>this.mapService.getFeatureTile(tileKey),
                     styleData.featureLayerStyle,
                     true,
-                    feature.id);
+                    resolvedFeature.peek((f: Feature) => f.id()));
```

Comment:
We still need to change the interface of TileVisualization on the C++ side for this to work.